packages/trader/src/App/Components/Elements/ContractAudit/contract-audit-item.jsx

```diff
@@ -37,6 +48,7 @@ ContractAuditItem.propTypes = {
     value: PropTypes.PropTypes.oneOfType([
         PropTypes.number,
         PropTypes.string,
+        PropTypes.node,
```

Comment:
In what case does `value` gets passed as `node`?

Reply:
a React component